### ISSUE TITLE
Rvm not loaded, aborted.

### ISSUE THREAD

**Issue by amolpujari** [rvm/rvm#3731]
I am constantly getting this issue on one of our servers, and I find no clue how to fix it.
I fixed it a couple of times by restarting out server, but now that solution is not working any more.
any clue?

I do have the following

```
RVM PATH line found in /home/deploy/.profile /home/deploy/.bashrc /home/deploy/.zshrc.
RVM sourcing line found in /home/deploy/.bashrc /home/deploy/.bash_profile /home/deploy/.zlogin.
```

Its Ubuntu 14.04 Rackspace, it was all working for few years, and not suddenly it starts throwing this error on one of our servers.

**Comment by pkuczynski** [comment]
Had similar issue after installing one of the newer version of RVM. The way I solved it was open every profile configuration file in my home directory (eg: `.profile`, `.bash_profile`, `.bashrc`) and remove every trace or RVM from there.

Then close terminal window, open it again and run

```
\curl -sSL https://get.rvm.io | bash -s stable --auto-dotfiles
```

After closing terminal and re-opening again, the issue was solved.
